runscripts/objWrite.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import networkx as nx
+
+"""
+write networkxgraph of skeleton to a wavefront obj file - primarily written
+for netmets (comparison of 2 networks)
+Link to the software - http://stim.ee.uh.edu/resources/software/netmets/
+Currently used as an input for visualization team
+"""
+
+
+def _removeEdgesInVisitedPath(subGraphSkeleton, path, isCycle):
+    """
+    Remove edges belonging to "path" from "subGraphSkeleton"
+    Parameters
+    ----------
+    subGraphSkeleton : Networkx graph
+        input graph to remove edges from
+
+    path : list
+       list of nodes in the path
+
+    isCycle : boolean
+       Specify if path is a cycle or not
+
+    Returns
+    -------
+    subGraphSkeleton : Networkx graph
+        graph changed inplace with visited edges removed
+
+    Notes
+    ------
+    given a visited path in variable path, the edges in the
+    path are removed in the graph "subGraphSkeleton".
+    if isCycle = 1 , the given path belongs to a cycle,
+    so an additional edge is formed between the last and the
+    first node to form a closed cycle/ path and is removed
+    """
+    shortestPathEdges = []
+    if isCycle == 0:
+        for index, item in enumerate(path):
+            if index + 1 != len(path):
+                shortestPathEdges.append(tuple((item, path[index + 1])))
+        subGraphSkeleton.remove_edges_from(shortestPathEdges)
```

Comment:
It looks like the only difference in the two for-loops is the else in the second one. Could it be written more compactly?

 +        for index, item in enumerate(path):
 +            if index + 1 != len(path):
 +                shortestPathEdges.append(tuple((item, path[index + 1])))
                   elif isCycle == 0:
 +        subGraphSkeleton.remove_edges_from(shortestPathEdges